pad_type: ps4
num_of_buttons: 13 
axis_linear_x: 1
axis_linear_y: 0
axis_angular: 2
axis_linear_z: 5
scale_linear: 1
scale_angular: 1
scale_linear_z: 20.0 
button_speed_up: 3
button_speed_down: 1
button_safety_override: 4
button_laser_mode: 6
button_dead_man: 5
button_dead_man_unsafe: 7
button_output_1: 2
button_output_2: 0
output_1: 10 
output_2: 11
button_home: 7 # unused
tilt_increment: 5
pan_increment: 5
zoom_increment: 300
button_kinematic_mode: 7 # unused
joystick_dead_zone: "true" #useful for x-wam
cmd_service_home:  rb2_controller/home
check_message_timeout: true
axis_elevator: 10
elevator_service_name: robotnik_base_control/set_elevator
laser_modes: ["standard","docking_station","cart","cart_docking_cart","docking_cart"]
